### ISSUE TITLE
Improve table accessibility

### ISSUE THREAD

**Issue by trilodge** [SAP/fundamental#1365]
> Is this issue related to a specific component?
table control 

> What version of the Fiori Fundamentals are you using?
1.4.9.

**Describe the solution you'd like**

The table component is missing some essential html attributes to connect rows and cols with their table headers and descriptions to make it a real data table.
I would suggest adding the "scope" attribute on `<th>` elements for `<thead>` and `<tbody>` with values of "row" or "col" to connect these locally for screenreaders etc. Also it would be nessessary to be able to put th inside tbody as well like given in the example in the link below.

**Additional context**

Markup and further explanation could be found here: https://inclusive-components.design/data-tables/

I already had a look into the table.scss file. It would need some restructuring to support this, because `.fd-table__fixed-col` would break the layout there.
